components/AvalancheCenterForecastZonePolygons.tsx

```diff
@@ -1,6 +1,5 @@
 import React from 'react';
 
-import {ActivityIndicator, Text, View} from 'react-native';
```

Comment:
Nice! Does the check also automatically remove these for us?

Reply:
It does not :(. It's possible that an editor plugin might!